index.js

```diff
@@ -76,11 +76,17 @@ async function fetchImage(url) {
   }
 }
 
-async function fetchImageId(url) {
+async function fetchImageIds(urls) {
   try {
-    const image = await fetchImage(url);
-    const id = await postImage(image);
-    return id;
+    const ids = await Promise.all(
+      urls.map(async (url) => {
+        const image = await fetchImage(url);
+        const id = await postImage(image);
+        return id;
```

Comment:
I would leave it inside `fetchImageId` function and reuse it here. Looks a bit better to read the code =]